### PR TITLE
update README instructions

### PR DESCRIPTION
**Pull request by Ssukriti**
The PR updates examples/utils to use function with caikit version. Adds instructions to README for development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,23 @@ Currently causal language models and sequence-to-sequence models are supported.
 #### Notes
 
 - The data model for text generative capabilities is baked into this repository itself at `caikit_nlp/data_model/generation.py`.
+
+## Development guidelines
+
+To contribute code to this repository, kindly fork the repository and create [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) with your contribution. For the contribution to be reviewed, we require unit tests for every new functionality added. We have some checks in place for code formatting and linting. We recommend that contributors ensure checks pass locally before opening pull requests for review. The following commands can be used for assistance:
```
Comment (gkumbhat) [inline README.md:15]: Some of this information is available [here](https://github.com/caikit/caikit-nlp/blob/main/CONTRIBUTING.md)

File: README.md [hunk 2]
```diff
@@ -9,3 +9,23 @@ Currently causal language models and sequence-to-sequence models are supported.
...
+
+### Useful commands
+
+1. Build the environment
```
Comment (gkumbhat) [inline README.md:19]: @evaline-ju any further recommendation on the commands?

File: README.md [hunk 3]
```diff
@@ -9,3 +9,23 @@ Currently causal language models and sequence-to-sequence models are supported.
 #### Notes
 
 - The data model for text generative capabilities is baked into this repository itself at `caikit_nlp/data_model/generation.py`.
+
+## Development guidelines
+
+To contribute code to this repository, kindly fork the repository and create [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) with your contribution. For the contribution to be reviewed, we require unit tests for every new functionality added. We have some checks in place for code formatting and linting. We recommend that contributors ensure checks pass locally before opening pull requests for review. The following commands can be used for assistance:
+
+### Useful commands
```
Comment (gkumbhat) [inline README.md:17]: Hi @hickeyma, currently we are missing this type of information from our projects, what would be the best way you would recommend to get this documented? 

Shall we start with adding this information in the `README` ?

File: README.md (hunk 1 above)
Comment (Ssukriti) [reply README.md:15]: oh wait I did not see that and didnt realize the overlap, I think I will move all thsi from README to contributing (only the commands that are missing )

**Comment by Ssukriti** [comment]
@gkumbhat I think most of the information is captured some way in the contributing file that you linked. I had missed looking at that. so we can close this PR I think and if needed we can link CONTRIBUTING.md from README .

**Pull request closed without merging**